CRAYSAT-1591: Document `list-vars`, refer to `docs-csm` for default files (#45)
Cray-HPE/docs-sat: docs/sat_bootprep.md

#### Viewing HPC CSM Software Recipe Variables

HPC CSM Software Recipe variables are available, and you can use them in the

> values of the keys listed above. A listing of the available variables and their
> values can be viewed by running `sat bootprep list-vars`. See `sat-bootprep(8)`
> for more information on options that may be used with the `list-vars`
> subcommand.

Reviewer: Why does `sat-bootprep(8)` include a hyphen, but `sat bootprep list-vars` does not? Just want to confirm this is correct. Also, I have a few wording suggestions below:

```suggestion
values of the keys listed above. View these variables and their values
by running `sat bootprep list-vars`. For more information on options that
can be used with the `list-vars` subcommand, see `sat-bootprep(8)`.
```

Reviewer: I like this rewording...let's go with that.

Default bootprep input files are delivered by the HPC CSM Software Recipe
product. You can access these files by cloning the `hpc-csm-software-recipe`

> repository, as described in the **Accessing `sat bootprep` files** section of

Reviewer: ```suggestion
repository, as described in the **Accessing `sat bootprep` files** process of
```

the [*Cray System Management

> Documentation*](https://cray-hpe.github.io/docs-csm/). The default input files
> can be found in the `bootprep` directory of the cloned repository:

Reviewer: ```suggestion
Documentation*](https://cray-hpe.github.io/docs-csm/). Find the
default input files in the `bootprep` directory of the cloned repository:
```

product. You can access these files by cloning the `hpc-csm-software-recipe`
repository.

To do this, follow steps 1-7 of the procedure described in the **Accessing `sat
bootprep` files** section of the [*Cray System Management
Documentation*](https://cray-hpe.github.io/docs-csm/). The default input files

> can be found in the `bootprep` directory of the cloned repository:

Reviewer 2: I would consider just condensing this paragraph into the previous one by adding "as described in the ... section of the Cray System Management Documentation." I don't feel like saying the steps is necessary here either as the number of steps could change, and then we'd be out of date here.
